### ISSUE TITLE
Improve AppleScript syntax highlighting

### ISSUE THREAD

**Issue by ylian** [HappenApps/Quiver#66]
AppleScript comments can start with `#`, which is equivalent to `--`. Need to add that into the "mode-applescript.js".

``` js
// In this.createKeywordMapper()
{token:"comment",regex:"#.*$”}
// and this:
this.lineCommentStart=“#”
```

**Comment by ylian** [comment]
Fixed in 2.1.3b5.

https://dl.dropboxusercontent.com/u/3445731/Quiver%202.1.3b5.zip

**Comment by ylian** [comment]
This is implemented in Quiver 3.0, released today.
